roman_numeral_analysis_tool.ily

```diff
@@ -313,12 +313,22 @@ its string, otherwise @code{#t}."
 #(define (make-figure-markup font-size)
    `(("f" . ,(make-general-align-markup Y DOWN
                (make-fontsize-markup font-size (make-flat-markup))))
+     ("ff" . ,(make-general-align-markup Y DOWN
+               (make-fontsize-markup font-size (make-doubleflat-markup))))
      ("flat" . ,(make-general-align-markup Y DOWN
                   (make-fontsize-markup font-size (make-flat-markup))))
+     ("flatflat" . ,(make-general-align-markup Y DOWN
+               (make-fontsize-markup font-size (make-doubleflat-markup))))
      ("s" . ,(make-general-align-markup Y -0.6
                (make-fontsize-markup font-size (make-sharp-markup))))
```

Comment:
The -1.9 introduces an inconsistency with the value of DOWN used make-accidental-markup.  You can see this with something like \markup \rN { xI x6 }

At some point these ought to be reconciled.